tests/Umbraco.Tests.AcceptanceTest/tests/ApiTesting/PartialView/PartialViewFolder.spec.ts

```diff
@@ -36,7 +36,7 @@ test.describe('Partial View Folder tests', () => {
   test('can add a partial view folder in another', async ({umbracoApi}) => {
     // Arrange
     const childFolderName = 'childFolder';
-    partialViewFolderPath = await umbracoApi.partialView.createFolder(partialViewFolderName);
+    await umbracoApi.partialView.createFolder(partialViewFolderName);
```

Comment:
Since we don't update the value for partialViewFolderPath, would the value not be empty when used in the tests?